app/src/main/java/ca/fuwafuwa/kaku/Windows/CaptureWindow.java

```diff
@@ -56,7 +57,13 @@ public boolean onTouch(MotionEvent e) {
     @Override
     public boolean onDoubleTap(MotionEvent e) {
         tessStarted = true;
-        mTessRunnable.runTess(new BoxParams(params.x, params.y + getStatusBarHeight(), params.width, params.height));
+
+        int[] viewPos = new int[2];
+        mWindowBox.getLocationOnScreen(viewPos);
+
+        // TODO: Replace the first 1 with R.drawable.border_translucent.StrokeWidth when I figure out how
+        int offset = KakuTools.dpToPx(this.context, 1)+1;
+        mTessRunnable.runTess(new BoxParams(viewPos[0]+offset, viewPos[1]+offset, params.width-(2*offset), params.height-(2*offset)));
```

Comment:
I acknowledge the existence of the bug you are mentioning, I did not consider it because the code was not in that state. However, I do not agree with what I am understanding as the end goal of having the OCR on the action up.

So if I were to move the rect from the corner to where I want it, it would run the OCR. But if I goofed and am off by a couple pixels so that half another kanji is in the rect I would have to move it again... which would trigger the OCR. Then I need to resize the window. But it is a small area so I cant see it all that well under my finger, so it takes me 2 or 3 tries. Now we are at 5+ OCR calls which take time and effort before I can align the box once.

This isn't a hypothetical, it is what has happened dozens of times while testing in games which have smaller font sizes and are prone to failure if partial kanji are shown.

If the OCR time was negligible, and the screenshots weren't saved to disc, and the OCR processes did not introduce any increased battery drain or processor load, then It would not bother me (in fact... I think it is a good idea). But as it stands now, that is not the case.